packages/atomic/src/components/search/tabs/atomic-tab-manager/e2e/atomic-tab-manager.e2e.ts

```diff
@@ -1,196 +1,198 @@
 import {test, expect} from './fixture';
 
-test.beforeEach(async ({tabManager}) => {
-  await tabManager.load();
-  await tabManager.hydrated.waitFor();
-});
-
-test.describe('when viewport is large enough to display all tabs', () => {
-  test('should be A11y compliant', async ({makeAxeBuilder}) => {
-    const accessibilityResults = await makeAxeBuilder().analyze();
-    expect(accessibilityResults.violations).toEqual([]);
+test.describe('AtomicTabManager', () => {
```

Comment:
It's preferable to have a 'top-level' describe.

Reply:
Agree !